auth_app/models.py

```diff
@@ -35,3 +39,24 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return "@{}".format(self.username)
+
+
+@receiver(post_save, sender=User)
+def after_user_signed_up(sender, instance, **kwargs):
+    if Bot.objects.filter(user=instance,).count() == 0:
+        Bot.objects.create(
+            name=instance.username,
+            token=generate_token(instance.username),
+            user=instance,
+        )
+
+
+class Bot(models.Model):
+    name = models.CharField(max_length=30)
+    token = models.CharField(max_length=200, default='')
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name='user',
+        null=True
```

Comment:
Why a bot could have a user as null?

Reply:
If I set null=False, now it asks me for a default value but I don't want a default value.